tutormfe/patches/caddyfile

```diff
@@ -3,5 +3,7 @@
     request_body {
         max_size 2MB
     }
+    encode gzip
+    reverse_proxy /api/mfe_config/v1* lms:8000
```

Comment:
Also this should be removed `encode gzip` becuse it's part of tutor anyway

Reply:
Good call: [there it is](https://github.com/overhangio/tutor/blob/master/tutor/templates/apps/caddy/Caddyfile#L26).  It's safe to remove `encode gzip`.